napari/utils/io.py

```diff
@@ -97,7 +98,17 @@ def imsave_tiff(filename, data):
     if compression_instead_of_compress:
         # 'compression' scheme is more complex. See:
         # https://forum.image.sc/t/problem-saving-generated-labels-in-cellpose-napari/54892/8
-        tifffile.imwrite(filename, data, compression=('zlib', 1))
+
+        imagecodecs_available = (
+            importlib.util.find_spec('imagecodecs') is not None
+        )
+
+        if data.dtype == bool and not imagecodecs_available:
+            tifffile.imwrite(filename, data)
+        else:
+            # boolean data needs special compression scheme
+            compression = 'zlw' if data.dtype == bool else ('zlib', 1)
```

Comment:
```suggestion
            compression = 'lzw' if data.dtype == bool else ('zlib', 1)
```
pretty sure it's LZW...

Reply:
That was an embarrassing mistake... I copy pasted the same line twice when I made my test (using a float array), so that's why I did not see any error... :/